### ISSUE TITLE
Music stops after 12 minutes

### ISSUE THREAD

**Issue by tyrann0us** [openopus-org/concertino_player#3]
I know that you @adrianosbr are aware of this bug, yet I wanted to create an issue that I can follow.

From https://github.com/openopus-org/concertino_player/issues/1#issuecomment-639982595:
>[…] MusicKit JS can’t play tracks longer than 15 minutes without halting. I reported
this bug about a year ago and got no answer at all.

In my experience, it stops after 12 minutes. Example tracks: https://concertino.app/u/7744/1452135651/1, movements 1 and 4.

**Comment by adrianosbr** [comment]
Yes, that's an old pain. MusicKit JS bug, even the official Apple Music web
player suffers with that. Concertino has to stop every track at minute 12
and make it resume after 30 or 40 seconds. If it waits till minute 15, it
crashes helplessly.

Apple has known about that since last year :/
